invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/form.overrides

```diff
@@ -60,14 +56,32 @@
         position: relative;
         padding-top: 3.5rem;
 
-        .ui.icon.button {
+        .ui.icon.button.close-btn {
           position: absolute;
           top: 1rem;
           right: 1.3rem;
+          margin-top: 0;
+        }
+      }
+    }
+
+    .additional-descriptions {
+      margin-top: -1rem;
```

Comment:
is it possible to do without negative margins ? they create a lot of issues, especially when used with SUI